thinc/backends/cupy_ops.py

```diff
@@ -150,8 +150,8 @@ def backprop_clipped_linear(
     ):
         if X.dtype == dY.dtype and X.dtype in ("float32", "float64"):
             return _custom_kernels.backprop_clipped_linear(
-                dY=dY,
-                X=X,
+                dY,
+                X,
                 slope=slope,
```

Comment:
Should the `slope`, `offset`, `min_val` and `max_val` become keyword-only?

Reply:
This PR makes them. Or do you mean in `CupyOps`?